js/library.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-env browser */
+
+let myLibrary = JSON.parse(window.localStorage.getItem('library'));
+if (myLibrary == null) {
+  myLibrary = [];
+}
+
+function Book(title, author, pages, readAlready) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.readAlready = readAlready;
+}
+
+Book.prototype = {
+  constructor: Book, // define the constructor property
+  updateReadStatus() {
+    if (this.readAlready === 'been read') {
+      this.readAlready = 'not been read';
+    } else {
+      this.readAlready = 'been read';
+    }
+  },
+};
+
+myLibrary.forEach((book) => { Object.setPrototypeOf(book, Book.prototype); });
+
+function render() {
+  document.getElementById('books-list').innerHTML = '';
+  myLibrary.forEach((book, index) => {
+    document.getElementById('books-list').innerHTML += `
+
+   <div class="col-12 col-md-3 col-lg-4">
+      <ul>
+        <li id="title"><b>Title</b>: ${book.title}</li>
+        <li id="author"><b>By</b>: ${book.author}</li>
+        <li id="pages"><b>No of Pages</b>: ${book.pages}</li>
+        <li id="read"><b>Has</b> ${book.readAlready}</li>
+        <button onclick="updateReadStatus(this)" data-attributes = ${index} >Update Read Status</button>
+        <button onclick="removeBookFromLibrary(this)" data-attributes = ${index} >Delete</button>
+      </ul>
+    </div>`;
+  });
+}
+
+function updateLocalStorage(array) {
+  window.localStorage.setItem('library', JSON.stringify(array));
+}
+
+// eslint-disable-next-line no-unused-vars
+function addBookToLibrary(ev) {
+  const title = document.getElementById('book_title').value;
+  const author = document.getElementById('book_author').value;
+  const pages = document.getElementById('pages').value;
+  let readAlready = document.getElementsByName('readAlready')[0].checked;
+  if (readAlready === true) {
+    readAlready = 'been read';
+  } else {
+    readAlready = 'not been read';
+  }
+
+  if (title === '' || author === '' || pages === '') {
+    alert('Please fill in all fields'); // eslint-disable-line no-alert
```

Comment:
Instead of disabling this rule with in-line comment you may consider putting an alert using js and dom manipulation, like a hide/show.

Instead of disabling all rules that give you problems you should address them.

This would make you a much better JS developer. 😄